ush/wave_grib2_sbs.sh

```diff
@@ -189,23 +189,44 @@ if [[ ! -s "${com_dir}/${outfile}.idx" ]]; then
   fi
 
   # Create index
-  ${WGRIB2} -s "${com_dir}/${outfile}" > "${com_dir}/${outfile}.idx"
+  ${WGRIB2} -s "${outfile}" > "${outfile}.idx"
 
-  # Create grib2 subgrid is this is the source grid
+  # Create grib2 subgrid if this is the source grid
   if [[ "${grdID}" = "${WAV_SUBGRBSRC}" ]]; then
     for subgrb in ${WAV_SUBGRB}; do
       subgrbref=$(echo ${!subgrb} | cut -d " " -f 1-20)
       subgrbnam=$(echo ${!subgrb} | cut -d " " -f 21)
       subgrbres=$(echo ${!subgrb} | cut -d " " -f 22)
       subfnam="${WAV_MOD_TAG}.${cycle}${ENSTAG}.${subgrbnam}.${subgrbres}.f${FH3}.grib2"
-      ${COPYGB2} -g "${subgrbref}" -i0 -x "${com_dir}/${outfile}" "${com_dir}/${subfnam}"
-      ${WGRIB2} -s "${com_dir}/${subfnam}" > "${com_dir}/${subfnam}.idx"
+      ${COPYGB2} -g "${subgrbref}" -i0 -x "${outfile}" "${subfnam}"
+      ${WGRIB2} -s "${subfnam}" > "${subfnam}.idx"
    done
   fi
 
   # 1.e Save in /com
+  # Move grib files to COM directory
+  if [[ -s "${outfile}" ]] && [[ -s "${outfile}.idx" ]]; then
+    cp -f "${outfile}" "${com_dir}/${outfile}"
+    cp -f "${outfile}.idx" "${com_dir}/${outfile}.idx"
```

Comment:
```suggestion
    cpfs "${outfile}" "${com_dir}/${outfile}"
    cpfs "${outfile}.idx" "${com_dir}/${outfile}.idx"
```
Lets do this right with using `cpfs`.